Modules/Transformation/src/BSplineFreeFormTransformationStatistical.cc

```diff
@@ -42,8 +42,8 @@ ::BSplineFreeFormTransformationStatistical()
 BSplineFreeFormTransformationStatistical
 ::BSplineFreeFormTransformationStatistical(const ImageAttributes    &attr,
                                            CPStatus              ****status,
-                                           const Matrix             &mat,
-                                           const Vector             &vec)
+                                           const mirtk::Matrix      &mat,
```

Comment:
Regarding the `init-dof` alternative: I believe the `init-dof` tool should output an FFD or DOF. An  SDM file is **not** and FFD nor a DOF. It is the concatenation of attributes, a list of control point status, a matrix and a vector:

```
// -----------------------------------------------------------------------------
void BSplineFreeFormTransformationStatistical::WriteSDM(const char *file)
{
  Cofstream to;
  to.Open(file);

  // Write header with file format version information
  to.WriteAsChar("SDM", 8);
  to.WriteAsChar("v1.0",    5);

  // Write lattice attributes
  to.WriteAsInt(&_attr._x, 1);
  to.WriteAsInt(&_attr._y, 1);
  to.WriteAsInt(&_attr._z, 1);

  to.WriteAsDouble(_attr._xaxis, 3);
  to.WriteAsDouble(_attr._yaxis, 3);
  to.WriteAsDouble(_attr._zaxis, 3);

  to.WriteAsDouble(&_attr._dx, 1);
  to.WriteAsDouble(&_attr._dy, 1);
  to.WriteAsDouble(&_attr._dz, 1);

  to.WriteAsDouble(&_attr._xorigin, 1);
  to.WriteAsDouble(&_attr._yorigin, 1);
  to.WriteAsDouble(&_attr._zorigin, 1);

  to.WriteAsInt(reinterpret_cast<const int *>(_CPStatus[0][0][0]),
                3 * _attr._x * _attr._y * _attr._z);

  // Write statistical deformation model data
  to << _BasisVectors << _MeanVector;
}
```

Given this i think the new tool is needed, because is very specific to SDMs.

With respect to the namespace, I was getting compilation errors:
`no known conversion for argument 4 from ‘mirtk::Vector’ to ‘const Vector& {aka const mirtk::Vector3D<double>&}’`
hence the need to add the namespace in front of Vector. I added it in front of Matrix for stylistic and consistency purposes, but can get rid of it if needed.